lighthouse-plugin-publisher-ads/audits/ad-render-blocking-resources.js

```diff
@@ -22,9 +22,9 @@ const {isAdTag} = require('../utils/resource-classification');
 const {URL} = require('url');
 
 const UIStrings = {
-  title: 'Minimal render-blocking resources found',
-  failureTitle: 'Avoid render-blocking resources',
-  description: 'Render-blocking resources slow down tag load times. Consider ' +
+  title: 'Minimal synchronous resources found',
+  failureTitle: 'Avoid synchronous resources',
+  description: 'Synchronous resources slow down tag load times. Consider ' +
```

Comment:
Maybe we should include something to still indicate that these are render-blocking: `Synchronous resources slow down tag load times and block ad rendering.`